controllers/index.js

```diff
@@ -150,6 +152,23 @@ function loadPrs(github, username) {
     return promise;
 }
 
+function loadPrsOpened(github) {
+    return new Promise((resolve, reject) => {
+        github.search.issues({
+            q: 'type:pr+label:hacktoberfest+created:>2018-09-30+created:<2018-11-01'
```

Comment:
*Any* PR opened in October counts, so don't think we need the label here. Also, can you use the times as well, as done here https://github.com/jenkoian/hacktoberfest-checker/blob/master/controllers/index.js#L130 just to keep in consistent.

Reply:
@jenkoian Also, it's still slightly misleading in a way that not all the users that have created PRs have signed up for Hacktoberfest as well. Only if we can find that out as well... :smiley:

Reply:
Yeah, I wrote about that in [the issue](https://github.com/jenkoian/hacktoberfest-checker/issues/281) we can't really get that info, so just a useful barometer more than number of participants, maybe we should add some wording to explain?

Reply:
I'll fix them.

And I agree in giving some explanation. Any ideias?

Reply:
I don't think this approach will work. I'm getting a lot of timeouts (504) because the number of PRs increase since yesterday. I'll seach another alternative for it.

Reply:
The issue isn't the rate limit but the time to process the request. It's return error 504.

Reply:
Hmm, not sure why that would happen. However, if you tweak the query to take out the hacktoberfest label, you get millions upon millions, so perhaps not that useful. It may be useful if we can do PRs opened by unique users, but not sure how easy/feasible that would be.